modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughHttpSender.java

```diff
@@ -389,24 +389,25 @@ private void sendRequestContent(final MessageContext msgContext) throws AxisFaul
 						return;
 					}
 
-					if ((disableChunking == null || !"true".equals(disableChunking)) ||
+                    if ((msgContext.getProperty(PassThroughConstants.FORCE_POST_PUT_NOBODY) != null &&
+                            (Boolean) msgContext.getProperty(PassThroughConstants.FORCE_POST_PUT_NOBODY))) {
```

Comment:
Shall we use Boolean.parseBoolean() instead of explicit casting? then no need to check the null as well. WDYT ?